src/palette/internal/palette/palettecreator.cpp

```diff
@@ -1423,6 +1423,7 @@ PalettePanel* PaletteCreator::newLinesPalettePanel()
 
     ottava = makeElement<Ottava>(gscore);
     ottava->setOttavaType(OttavaType::OTTAVA_22MB);
+    ottava->setPlacement(Placement::BELOW);
```

Comment:
May not be needed, certainly not for the issue at hand, but I found this to be pretty inconsistent, at least.